### ISSUE TITLE
How can I modify the mask loss to discourage multi-part mask output?

### ISSUE THREAD

**Issue by wuyuanyi135** [facebookresearch/Detectron#438]
### Expected results
I am training the mask rcnn network with some microscopic images. Unlike the common dataset, the annotated regions are small and blurry. Although all the training information I provide is in one piece, the inference output still show many "multi-part" masks.  I wonder, whether it is possible or not, to modify the loss function to prevent the network to output any multi-part objects?

### Actual results

Here attach an example inference output and I labelled all the multipart outputs with red arrows.
![inf](https://user-images.githubusercontent.com/11760870/40274065-ce583bf8-5b9b-11e8-954e-4d37f14b2170.png)

Thank you!

**Comment by javi-clear-image-ai** [comment]
I'm not a pro, but I guess you just need to use another net... 
In any case there are many other options that you could tune in the training and inference steps, like the bboxes size, the learning rate, the threshold scores, epocs, etc
